contentcuration/contentcuration/frontend/channelEdit/utils.js

```diff
@@ -252,6 +345,8 @@ export function getCompletionCriteriaLabels(node) {
       labels.completion = metadataStrings.$tr('completeDuration');
       if (suggestedDuration) {
         labels.duration = secondsToHms(suggestedDuration);
+      } else {
+        getAudioVideoDefaultDuration(node);
```

Comment:
May be related to a comment I left in the ResourcePanel file - it'd be good to have the new behavior covered by a unit test.

Could be done here or as a follow-up

Reply:
One clarification question and perhaps a suggestion, this as a whole

```js
case CompletionCriteriaModels.TIME:
  labels.completion = metadataStrings.$tr('completeDuration');
  if (suggestedDuration) {
    labels.duration = secondsToHms(suggestedDuration);
  } else {
    getAudioVideoDefaultDuration(node);
  }
  break;
```

says that if there's no suggested duration set for the time completion criteria model, we should use audio/video default duration. So it seems that we might have a situation when we use audio/video default duration values for content that's actually not audio/video. So I wonder if it'd make sense to check if a node is audio/video before using audio/video default duration (similarly to what you did in the ResourcePanel)?

Reply:
Oh, one thing I noticed right now after discussing files with you above is that here you call `getAudioVideoDefaultDuration(node)` but as you mentioned, the function needs files, not a node. So this will need to be sorted out in one way or another. Feel free to organize it as you wish, I think the only requirement is that the expected parameter value matches.